PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -341,8 +341,9 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
         miniPFRelIso_all = Var("userFloat('miniIsoAll')/pt",float,doc="mini PF relative isolation, total (with scaled rho*EA PU Winter22V1 corrections)"),
         pfRelIso03_chg = Var("userFloat('PFIsoChg')/pt",float,doc="PF relative isolation dR=0.3, charged component"),
         pfRelIso03_all = Var("userFloat('PFIsoAll')/pt",float,doc="PF relative isolation dR=0.3, total (with rho*EA PU Winter22V1 corrections)"),                
-        jetRelIso = Var("?userCand('jetForLepJetVar').isNonnull()?(1./userFloat('ptRatio'))-1.:userFloat('PFIsoAll04')/pt",float,doc="Relative isolation in matched jet (1/ptRatio-1, pfRelIso04_all if no matched jet)",precision=8),        
+        jetPtRatio = Var("?userCand('jetForLepJetVar').isNonnull()?userFloat('ptRatio'):-1.",float,doc="ratio of electron pt to the associated jet pt (-1. if none)",precision=10),
```

Comment:
It seems that `PFIsoAll04` is not stored in NanoAOD, so if we make this change we will need to add it.

Reply:
Good catch, @hqucms. I assumed that was included in the same way as it it for muons.

But actually,  there is somewhat an inconsistency as of now.  `Electron_jetRelIso` takes  `PFIsoAll04` (for Run3)[https://github.com/cms-sw/cmssw/blob/dfc9a3cd0e7340aff5dfa94920edd6f738e8c5c5/PhysicsTools/NanoAOD/python/electrons_cff.py#L345] and 
 `PFIsoAll04_Fall17V2` [for Run2](https://github.com/cms-sw/cmssw/blob/dfc9a3cd0e7340aff5dfa94920edd6f738e8c5c5/PhysicsTools/NanoAOD/python/electrons_cff.py#L456), but what is used for [LepGood_jetPtRatio](https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/electrons_cff.py#L281) in the electronPROMPTMVA is `PFIsoAll04_Fall17V2` in all cases. I suppose that this is because the MVA training was done witth the `Fall17V2` version, but then `Electron_jetRelIso` is inconsistent with the MVA input for Run3. 
 I'm unsure on how to proceed then: it depends if the goal is to be able to recover the variable used as input in the `electronPROMPTMVA (then we need to add `PFIsoAll04_Fall17V2`), or to be able to recover the current `Electron_jetRelIso` (then we have to add `PFIsoAll04` for Run3 or `PFIsoAll04_Fall17V2` for Run2, but then I miss what's the point of having this variable).

Reply:
The current code suggests that one should use `PFIsoAll04` for Run3 and `PFIsoAll04_Fall17V2` for Run2, so I suppose when the `electronPROMPTMVA` is re-trained for Run3 it should move to using `PFIsoAll04`? 

Maybe @cms-sw/egamma-pog-l2 could comment and clarify on the plans for the update of `electronPROMPTMVA` for Run3?

Reply:
Added `pfRelIso04_all`.